eclass/distutils-r1.eclass

```diff
@@ -78,7 +78,28 @@ esac
 # to be exported. It must be run in order for the eclass functions
 # to function properly.
 
+# @ECLASS-VARIABLE: DISTUTILS_USE_PEP517
+# @PRE_INHERIT
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Enable experimental PEP 517 mode for the specified build system.
+# In this mode, the complete build and install is done
+# in python_compile(), venv-style install tree is provided
+# to python_test() and python_install() just merges the temporary
+# install tree into real fs.
+#
+# The variable specifies the build system used.  Currently,
+# the following values are supported:
+# - flit - flit_core backend
+# - setuptools - distutils or setuptools (incl. legacy mode)
+# - standalone - standalone build systems without external deps
+#                (used for bootstrapping).
```

Comment:
Could you run doc generation for eclass into man pages. I have a feeling we need extra newline between items in the list.